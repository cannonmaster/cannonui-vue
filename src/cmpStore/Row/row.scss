.chihuoui-row {
  width: 100%;
  overflow: hidden;
  &:after {
    display: block;
    height: 0;
    clear: both;
    visibility: hidden;
    content: '';
  }
  &-flex {
    display: flex;
    &:after {
      display: none;
    }
    .chihuoui-col {
      float: none;
    }
  }
  &-justify-center {
    justify-content: center;
  }
  &-justify-end {
    justify-content: end;
  }
  &-justify-space-between {
    justify-content: space-between;
  }
  &-justify-space-around {
    justify-content: space-around;
  }
  &-justify-space-evenly {
    justify-content: space-evenly;
  }
  &-align-flex-start {
    align-items: flex-start;
  }
  &-align-flex-end {
    align-items: flex-end;
  }
  &-align-flex-center {
    align-items: center;
  }
  &-align-flex-end {
    align-items: flex-end;
  }
  &-flex-wrap {
    flex-wrap: wrap;
  }
  &-flex-nowrap {
    flex-wrap: nowrap;
  }
  &-flex-reverse {
    flex-direction: row-reverse;
  }
}
