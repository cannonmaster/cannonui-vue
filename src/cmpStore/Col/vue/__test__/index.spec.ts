import Col from '../index';
import Row from '../../../Row/vue/index';
import { mount } from '@vue/test-utils';

test('should render Col and gutter', async () => {
  const wrapper = mount({
    components: {
      Row,
      Col,
    },
    template: `
        <Row gutter="6">
            <Col span="8">8</Col>
            <Col span="8">8</Col>
            <Col span="8">8</Col>
        </Row>
    `,
  });
  expect(wrapper.html()).toMatchSnapshot();
});
// // 右
// 1
// 2 30
// 3 24
// 0 45/44？
// -1 54？
// -2 66？
// -3
// // 左
// 0 48
// 3 26？
// -2 76

// //左
// 3 28
// 2 37
// //右
// 0 55？
// 2 34
// 3 29
// -1 68

// e2e
// //左
// 0 62
// -2 93
// //右
// -2 85
// 2 41
// 3 43
