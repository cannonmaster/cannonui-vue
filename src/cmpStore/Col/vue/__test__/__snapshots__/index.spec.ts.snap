// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Col and gutter 1`] = `
"<view class="chihuoui-row
        chihuoui-row-flex
        chihuoui-row-align-flex-center
        chihuoui-row-type-flex
        chihuoui-row-justify-center
        chihuoui-row-flex-wrap">
  <view class="chihuoui-col chihuoui-col-gutter chihuoui-col-offset-0 chihuoui-col-8" style="padding-left: 3px; padding-right: 3px;">8</view>
  <view class="chihuoui-col chihuoui-col-gutter chihuoui-col-offset-0 chihuoui-col-8" style="padding-left: 3px; padding-right: 3px;">8</view>
  <view class="chihuoui-col chihuoui-col-gutter chihuoui-col-offset-0 chihuoui-col-8" style="padding-left: 3px; padding-right: 3px;">8</view>
</view>"
`;
